contracts/sol6/Dao/IKyberStaking.sol

```diff
@@ -27,6 +27,40 @@ interface IKyberStaking is IEpochUtils {
 
     function withdraw(uint256 amount) external;
 
+    /**
+     * @notice return combine data (stake, delegatedStake, representative) of a staker
+     * @dev allow to get staker data up to current epoch + 1
+     */
+    function getStakerData(address staker, uint256 epoch)
+        external
+        view
+        returns (
+            uint256 stake,
+            uint256 delegatedStake,
+            address representative
+        );
+
+    function getStake(address staker, uint256 epoch) external view returns (uint256);
```

Comment:
I am thinking the 3 separate functions should not be here.

its for "advanced" users

if they want it.
they should write their own interface

Reply:
I don't see any downside having it... Developers are lazy ppl, asking them to do more work isn't nice =)